### DISCUSSION TITLE
Picture of Doorbell pixelated

### DISCUSSION THREAD

**Discussion by donwolle** [fuatakgun/eufy_security#196] [Q&A]
Hi together, 

thank you for your work :-)!

generally the addon works, but often my wired doorbell picture in Home Assistant ist pixelated, using your Addon. 
I set the streaming quality to low in the eufy app.

do you have any suggestions how to fix this problem?

![image](https://user-images.githubusercontent.com/96878967/147945622-0d7b3672-0243-4354-835d-0a9a016399b8.png)
![image](https://user-images.githubusercontent.com/96878967/147947489-9a6c7587-64ad-4e14-93e4-27ad00544ee9.png)
![image](https://user-images.githubusercontent.com/96878967/147947600-4271cbde-954a-46ba-9d1b-939d0cbb317e.png)
![image](https://user-images.githubusercontent.com/96878967/147947704-7e02d1aa-c37b-488c-ab96-335e647ba59d.png)
![image](https://user-images.githubusercontent.com/96878967/147947752-d5076536-d40e-45da-af4f-341132b247cc.png)
![image](https://user-images.githubusercontent.com/96878967/147947836-5f646d7c-7be4-44dd-92b7-d668dd20369b.png)
![image](https://user-images.githubusercontent.com/96878967/147947868-e6f6babf-40b6-4755-a1c1-0969129047e6.png)

**Comment by fuatakgun** [comment]
I suggest you to create an issue for this, rather than discussion, I am not good at looking topics here.

Anyway, is it pixelated while streaming or to show latest captured photo?

**Reply by donwolle** [reply]
sorry - first day at github ;-).

i have problems while streaming. 
Sometimes a "cam reboot" helps. 
But a few minutes later its pixelated again while streaming.

last captured photo is not pixelated.
stream in eufy app is also ok.

**Comment by fuatakgun** [comment]
set everything as low as storage quality, streaming quality, codec etc.
